update_compilers/install_libraries.sh

```diff
@@ -81,7 +81,7 @@ update_boost_archive
 ce_install 'libraries/c++/llvm'
 
 if install_nightly; then
```

Comment:
in theory the `install_nightly` is already accounted-for in the `ce_install` function in `common.inc`.

Hopefully once everything's moved over these scripts all get super simple (`ce_install install --enable nightly` to do everything) :)

Reply:
It is, but I just tried it out and it also does the normal installs, because those aren't being excluded.
The if: nightly are only being included. So it matters mostly how they are pathed in the yaml files.

So currently I can do:
```
if install_nightly; then
    ce_install 'libraries/c/nightly'
    ce_install 'libraries/c++/nightly'
    ce_install 'libraries/d/nightly'
    ce_install 'libraries/cuda/nightly'
else
    ce_install 'libraries/c'
    ce_install 'libraries/c++'
    ce_install 'libraries/d'
    ce_install 'libraries/cuda'
fi
```

I think in order to being able to only give 1 command I have to put nightly first in the yaml? so I can do `ce_install libraries/nightly`

Reply:
Ah! the nightlies need a `if: nightly` in the YAML probably. That's a way to "hide" them by default (which I did so that average users of ce_install don't get a giant bunch of compilers and libraries every time they run the command)